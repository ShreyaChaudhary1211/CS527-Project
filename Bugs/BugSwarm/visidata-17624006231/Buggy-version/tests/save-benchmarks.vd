sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Date		type-date		@	set type of current column to date
benchmark	Quantity		type-int		#	set type of current column to int
benchmark	Unit		type-currency		$	set type of current column to currency
