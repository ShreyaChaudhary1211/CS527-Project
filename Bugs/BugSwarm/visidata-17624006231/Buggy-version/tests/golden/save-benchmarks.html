<h2>benchmark</h2><table id="benchmark">
<tr><th>Date</th><th>Customer</th><th>SKU</th><th>Item</th><th>Quantity</th><th>Unit</th><th>Paid</th></tr>
<tr><td>2018-07-03</td><td>Robert Armstrong</td><td>FOOD213</td><td>BFF Oh My Gravy! Beef &amp; Salmon 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-07-03</td><td>Kyle Kennedy</td><td>FOOD121</td><td>Food, Adult Cat - 3.5 oz</td><td>1</td><td>4.22</td><td>$4.22</td></tr>
<tr><td>2018-07-05</td><td>Douglas &quot;Dougie&quot; Powers</td><td>FOOD121</td><td>Food, Adult Cat 3.5 oz</td><td>1</td><td>4.22</td><td>$4.22</td></tr>
<tr><td>2018-07-06</td><td>&#26716; &#39640;&#27211; (Sakura Takahashi)</td><td>FOOD122</td><td>Food, Senior Wet Cat - 3 oz</td><td>12</td><td>1.29</td><td>157&#165;</td></tr>
<tr><td>2018-07-10</td><td>David Attenborough</td><td>NSCT201</td><td>Food, Salamander</td><td>30</td><td>0.05</td><td>$1.5</td></tr>
<tr><td>2018-07-10</td><td>Susan Ashworth</td><td>CAT060</td><td>Cat, Korat (Felis catus)</td><td>1</td><td>720.42</td><td>$720.42</td></tr>
<tr><td>2018-07-10</td><td>Susan Ashworth</td><td>FOOD130</td><td>Food, Kitten 3kg</td><td>1</td><td>14.94</td><td>$14.94</td></tr>
<tr><td>2018-07-13</td><td>Wil Wheaton</td><td>NSCT523</td><td>Monster, Rust (Monstrus gygaxus)</td><td>1</td><td>39.95</td><td>$39.95</td></tr>
<tr><td>2018-07-13</td><td>Robert Armstrong</td><td>FOOD216</td><td>BFF Oh My Gravy! Chicken &amp; Shrimp 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-07-17</td><td>Robert Armstrong</td><td>FOOD217</td><td>BFF Oh My Gravy! Duck &amp; Tuna 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-07-17</td><td>Helen Halestorm</td><td>LAGO342</td><td>Rabbit (Oryctolagus cuniculus)</td><td>2</td><td>32.94</td><td>$65.88</td></tr>
<tr><td>2018-07-18</td><td>&#26716; &#39640;&#27211; (Sakura Takahashi)</td><td>FOOD122</td><td>Food, Senior Wet Cat - 3 oz</td><td>6</td><td>1.29</td><td>157&#165;</td></tr>
<tr><td>2018-07-19</td><td>Rubeus Hagrid</td><td>FOOD170</td><td>Food, Dog - 5kg</td><td>5</td><td>44.95</td><td>$224.75</td></tr>
<tr><td>2018-07-20</td><td>Jon Arbuckle</td><td>FOOD167</td><td>Food, Premium Wet Cat - 3.5 oz</td><td>50</td><td>3.95</td><td>$197.5</td></tr>
<tr><td>2018-07-23</td><td>Robert Armstrong</td><td>FOOD215</td><td>BFF Oh My Gravy! Lamb &amp; Tuna 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-07-23</td><td>Douglas &quot;Dougie&quot; Powers</td><td>TOY235</td><td>Laser Pointer</td><td>1</td><td>16.12</td><td>$16.12</td></tr>
<tr><td>2018-07-24</td><td>&#26716; &#39640;&#27211; (Sakura Takahashi)</td><td>FOOD122</td><td>Food, Senior Wet Cat - 3 oz</td><td>3</td><td>1.29</td><td>157&#165;</td></tr>
<tr><td>2018-07-26</td><td>Douglas &quot;Dougie&quot; Powers</td><td>FOOD420</td><td>Food, Shark - 10 kg</td><td>1</td><td>15.70</td><td>$15.7</td></tr>
<tr><td>2018-07-27</td><td>&#26716; &#39640;&#27211; (Sakura Takahashi)</td><td>FOOD122</td><td>Food, Senior Wet Cat - 3 oz</td><td>3</td><td>1.29</td><td>157&#165;</td></tr>
<tr><td>2018-07-30</td><td>&#26716; &#39640;&#27211; (Sakura Takahashi)</td><td>RETURN</td><td>Food, Senior Wet Cat - 3 oz</td><td>1</td><td>1.29</td><td>157&#165;</td></tr>
<tr><td>2018-07-31</td><td>Rubeus Hagrid</td><td>CAT060</td><td>Food, Dragon - 50kg</td><td>5</td><td>720.42</td><td>$3602.1</td></tr>
<tr><td>2018-08-01</td><td>David Attenborough</td><td>FOOD360</td><td>Food, Rhinocerous - 50kg</td><td>4</td><td>5.72</td><td>$22.88</td></tr>
<tr><td>2018-08-02</td><td>Susan Ashworth</td><td>CAT110</td><td>Cat, Maine Coon (Felix catus)</td><td>1</td><td>1309.68</td><td>$1309.68</td></tr>
<tr><td>2018-08-02</td><td>Susan Ashworth</td><td>FOOD130</td><td>Food, Kitten 3kg</td><td>3</td><td>14.94</td><td>$44.82</td></tr>
<tr><td>2018-08-06</td><td>Robert Armstrong</td><td>FOOD212</td><td>BFF Oh My Gravy! Beef &amp; Chicken 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-08-07</td><td>Juan Johnson</td><td>REPT082</td><td>Kingsnake, California (Lampropeltis getula)</td><td>1</td><td>89.95</td><td>$89.95</td></tr>
<tr><td>2018-08-07</td><td>Juan Johnson</td><td>RDNT443</td><td>Mouse, Pinky (Mus musculus)</td><td>1</td><td>1.49</td><td>$1.49</td></tr>
<tr><td>2018-08-10</td><td>Robert Armstrong</td><td>FOOD211</td><td>BFF Oh My Gravy! Chicken &amp; Turkey 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-08-13</td><td>Monica Johnson</td><td>RDNT443</td><td>Mouse, Pinky (Mus musculus)</td><td>1</td><td>1.49</td><td>$1.49</td></tr>
<tr><td>2018-08-13</td><td>Mar&#237;a Fern&#225;ndez</td><td>FOOD146</td><td>Forti Diet Prohealth Mouse/Rat 3lbs</td><td>2</td><td>2.00</td><td>$4.0</td></tr>
<tr><td>2018-08-15</td><td>Mr. Praline</td><td>RETURN</td><td>Parrot, Norwegian Blue (Mopsitta tanta)</td><td>1</td><td>2300.00</td><td>-$2300.0</td></tr>
<tr><td>2018-08-15</td><td>Kyle Kennedy</td><td>FOOD121</td><td>Food, Adult Cat - 3.5 oz</td><td>2</td><td>4.22</td><td>$8.44</td></tr>
<tr><td>2018-08-16</td><td>Helen Halestorm</td><td>RETURN</td><td>Rabbit (Oryctolagus cuniculus)</td><td>6</td><td>0.00</td><td>$0.0</td></tr>
<tr><td>2018-08-16</td><td>Kyle Kennedy</td><td>DOG010</td><td>Dog, Golden Retriever (Canis lupus familiaris)</td><td>1</td><td>2495.99</td><td>$2495.99</td></tr>
<tr><td>2018-08-16</td><td>Michael Smith</td><td>BIRD160</td><td>Parakeet, Blue (Melopsittacus undulatus)</td><td>1</td><td>29.95</td><td>$31.85</td></tr>
<tr><td>2018-08-17</td><td>Rubeus Hagrid</td><td>NSCT201</td><td>Food, Spider</td><td>5</td><td>0.05</td><td>$0.25</td></tr>
<tr><td>2018-08-20</td><td>Kyle Kennedy</td><td>RETURN</td><td>Dog, Golden Retriever (Canis lupus familiaris)</td><td>1</td><td>1247.99</td><td>-$1247.99</td></tr>
<tr><td>2018-08-20</td><td>&#1502;&#1512;&#1493;&#1505;&#1497;&#1492; &#1504;&#1497;&#1505;&#1504;&#1492;&#1493;&#1500;&#1509; &#1488;&#1489;&#1493;&#1500;&#1506;&#1508;&#1497;&#1492;</td><td>GOAT224</td><td>Goat, American Pygmy (Capra hircus)</td><td>1</td><td>499.00</td><td>$160.51</td></tr>
<tr><td>2018-08-20</td><td>Monica Johnson</td><td>NSCT201</td><td>Crickets, Adult Live (Gryllus assimilis)</td><td>30</td><td>0.05</td><td>$1.5</td></tr>
<tr><td>2018-08-20</td><td>David Attenborough</td><td>NSCT084</td><td>Food, Pangolin</td><td>30</td><td>0.17</td><td>$5.10</td></tr>
<tr><td>2018-08-21</td><td>Robert Armstrong</td><td>FOOD214</td><td>BFF Oh My Gravy! Duck &amp; Salmon 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-08-22</td><td>David Attenborough</td><td>BIRD160</td><td>Food, Quoll</td><td>1</td><td>29.95</td><td>$29.95</td></tr>
<tr><td>2018-08-22</td><td>Jon Arbuckle</td><td>FOOD170</td><td>Food, Adult Dog - 5kg</td><td>1</td><td>44.95</td><td>$44.95</td></tr>
<tr><td>2018-08-22</td><td>&#1502;&#1512;&#1493;&#1505;&#1497;&#1492; &#1504;&#1497;&#1505;&#1504;&#1492;&#1493;&#1500;&#1509;</td><td>SFTY052</td><td>Fire Extinguisher, kitchen-rated</td><td>1</td><td>61.70</td><td>$61.70</td></tr>
<tr><td>2018-08-24</td><td>Robert Armstrong</td><td>FOOD218</td><td>BFF Oh My Gravy! Chicken &amp; Salmon 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-08-27</td><td>Monica Johnson</td><td>NSCT443</td><td>Mealworms, Large (Tenebrio molitor) 100ct</td><td>1</td><td>1.99</td><td>$1.99</td></tr>
<tr><td>2018-08-28</td><td>Susan Ashworth</td><td>CAT020</td><td>Cat, Scottish Fold (Felis catus)</td><td>1</td><td>1964.53</td><td>$1964.53</td></tr>
<tr><td>2018-08-28</td><td>Susan Ashworth</td><td>FOOD130</td><td>Food, Kitten 3kg</td><td>2</td><td>14.94</td><td>$29.88</td></tr>
<tr><td>2018-08-29</td><td>Robert Armstrong</td><td>FOOD219</td><td>BFF Oh My Gravy! Chicken &amp; Pumpkin 2.8oz</td><td>4</td><td>12.95</td><td>$51.8</td></tr>
<tr><td>2018-08-31</td><td>Robert Armstrong</td><td>FOOD219</td><td>BFF Oh My Gravy! Chicken &amp; Pumpkin 2.8oz</td><td>144</td><td>12.95</td><td>$1864.8</td></tr>
<tr><td>2018-08-31</td><td>Juan Johnson</td><td>REPT217</td><td>Lizard, Spinytail (Uromastyx ornatus)</td><td>1</td><td>99.95</td><td>$99.95</td></tr>
</table>